src/layouts/Layout.astro

```diff
@@ -22,35 +22,62 @@ const { title, headerProps } = Astro.props;
     <meta name="viewport" content="width=device-width" />
     <link
       rel="icon"
-      href="/website-v2/favicon.svg"
+      href="/website-v2/footer/logo.png"
       sizes="any"
       type="image/svg+xml"
     />
     <meta name="generator" content={Astro.generator} />
     <title>{title}</title>
   </head>
   <body class="bg-white">
-    <nav
-      class="fixed left-0 top-0 z-50 w-full bg-black bg-opacity-[0.95] p-4"
-    >
-      <div class="m-auto flex w-full justify-between">
-        <a href="/website-v2">
-          <img
-            src="/website-v2/footer/logo.png"
-            class="h-16"
-          />
-        </a>
-        <Hamburger client:only />
-      </div>
-    </nav>
+    {
+      headerProps.isContain ? (
+        <div>
+          <nav class="fixed left-0 top-0 z-50 w-full bg-black bg-opacity-[0.5] p-4 backdrop-blur-sm">
+            <div class="m-auto flex w-full justify-between">
+              <a href="/website-v2">
+                <img
+                  src="/website-v2/footer/logo.png"
+                  class="h-16"
+                />
+              </a>
+              <Hamburger client:only />
+            </div>
+          </nav>
+        </div>
+      ) : (
+        <div>
+          <nav class="fixed left-0 top-0 z-50 w-full bg-black bg-opacity-[0.95] p-4">
+            <div class="m-auto flex w-full justify-between">
+              <a href="/website-v2">
+                <img
+                  src="/website-v2/footer/logo.png"
+                  class="h-16"
+                />
+              </a>
+              <Hamburger client:only />
+            </div>
+          </nav>
+        </div>
+      )
+    }
+
     <Notification
       message="Apply for a FOSS Career with Chaincode & Bitshala!"
       url="https://chaincode.applytojob.com/apply/40kEabYKUY/Bitshala-Start-Your-Career-In-Bitcoin-FOSS-Development"
       client:only
     />
-    <Header {...headerProps}>
-      <slot name="headerChild" />
-    </Header>
+    {
+      headerProps.isContain ? (
+        <div />
```

Comment:
Replace with `null`. Empty divs are not good.

Reply:
we can have only one div, we don't need to check in that case